Modules/Files/test/Files.API.Tests.Performance.EndToEnd/src/utils.ts

```diff
@@ -0,0 +1,73 @@
+import http from "k6/http";
+import exec from "k6/execution";
+
+export interface Configuration {
+  Host: string;
+  Client_Secret: string;
+  Username: string;
+  Password: string;
+  Size: string;
+}
+
+export interface Size {
+  vus: number;
+  iterations: number;
+}
+
+export function tomorrow(): Date {
+  const date = new Date();
+  date.setDate(date.getDate() + 1);
+
+  return date;
+}
+
+export function getAuthenticationHeader(configuration: Configuration): string {
+  const bodyConnectToken = {
+    client_id: "test",
+    client_secret: configuration.Client_Secret,
+    username: configuration.Username,
+    password: configuration.Password,
+    grant_type: "password",
+  };
+
+  const authToken = http
+    .post(`${configuration.Host}/connect/token`, bodyConnectToken, {
+      headers: {
+        "Content-Type": "application/x-www-form-urlencoded",
+      },
+    })
+    .json("access_token")!
+    .toString();
+
+  return `Bearer ${authToken}`;
+}
+
+export function getConfiguration(): Configuration {
+  if (!__ENV.HOST) {
+    exec.test.abort("Parameter 'HOST' cannot be null or empty");
+  }
+
+  if (!__ENV.CLIENT_SECRET) {
+    exec.test.abort("Parameter 'CLIENT_SECRET' cannot be null or empty");
+  }
+
+  if (!__ENV.USERNAME) {
+    exec.test.abort("Parameter 'USERNAME' cannot be null or empty");
+  }
+
+  if (!__ENV.PASSWORD) {
+    exec.test.abort("Parameter 'PASSWORD' cannot be null or empty");
+  }
+
+  if (!__ENV.SIZE) {
+    exec.test.abort("Parameter 'SIZE' cannot be null or empty");
+  }
```

Comment:
I would probably still add a method assertEnvVarExists, which you can use in this method. Just to reduce duplication.
```ts
assertEnvVarExists("HOST");
assertEnvVarExists("CLIENT_SECRET");
assertEnvVarExists("USERNAME");
assertEnvVarExists("PASSWORD");
assertEnvVarExists("SIZE");
```

Reply:
Ok, I implemented this method now. While testing this I noticed that when the USERNAME variable wasn't set, k6 was using  my username for some reason. It seems that USERNAME is already used as an environment variable by default (I think), so when we pass it as a parameter, it is overwritten, so I had to change it to USER instead.